rust/automerge-wasm/examples/cloudflare/src/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef */
+import * as Automerge from "../../../web";
```

Comment:
Slight nit, can we do `import * as Automerge from "@automerge/automerge"` here as that's what people will need to do in production. If you want to test that locally you can stand up an NPM registry with the latest JS package installed in it by doing `cd javascript/e2e; yarn install; yarn e2e run-registry` and then you can install the dependency in the example via `cd javascript/examples/cloudflare; yarn install --registry http://localhost:4873`

Reply:
Good look, this actually caught a bug. Fixed, done!